pkgs/by-name/ra/radicle-job/update.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p coreutils gnugrep common-updater-scripts nix-update
+
+version=$(list-git-tags | grep -oP '^releases/v\K\d+\.\d+\.\d+$' | sort -rV | head -1)
```

Comment:
How is that supposed to work? list-git-tags of the src?

Reply:
`fetchFromRadicle` simply wraps `fetchgit`, so it exposes a `url` attribute which is used by `list-git-tags`